### DISCUSSION TITLE
dwd-mosmix, HTML Tabelle, Anzeige Windrichtung in der Wettervorhersage des DWD

### DISCUSSION THREAD

**Discussion by hoetzgit** [roe-dl/weewx-DWD#9] [Q&A] [answered]
Hallo, ich habe eine Frage zur Ansicht der Windrichtung in der Wettervorhersage. Wenn ich es richtig verstanden habe, dann beziehen sich die Angaben des DWD in Grad bezogen auf die Himmelsrichtung Nord nach rechts drehend.  Wenn ich mir die Symbole ansehe, dann ist das Quellsymbol in `dwd-mosmix` die CSS-Klasse `wi-direction-down`.

Ist das so korrekt oder müsste es nicht eher `wi-direction-up` sein?

... oder ist es die Richtung, in die der Wind weht?

**Comment by roe-dl** [comment, marked as answer]
Das ist eine Frage der Betrachtung. Die Richtung, die der DWD angibt, ist, wo der Wind *herkommt*. Der Pfeil zeigt dagegen, wo der Wind *hingeht*. Zumindest würde ich einen Pfeil so interpretieren. So ist es jetzt gemeint: der Wind weht in Pfeilrichtung. Ein Nordwind hat also 0° und den Pfeil nach unten Richtung Süden.

**Comment by hoetzgit** [comment]
Vielen Dank, dann habe ich es verstanden.

**Comment by roe-dl** [comment]
Um Mißverständnisse auszuschließen, habe ich einen Tooltip bei den Pfeilen ergänzt, der Windrichtung und Gradzahl anzeigt, Commit ff408744932e55f911e9930fe9dd2f3697a4a4c8
